src/java.base/share/classes/module-info.java

```diff
@@ -160,6 +160,9 @@
         jdk.compiler,
         jdk.jlink;
     exports jdk.internal.logger to
+    /*[IF OPENJCEPLUS_SUPPORT]*/
+        openjceplus,
+    /*[ENDIF] OPENJCEPLUS_SUPPORT */
```

Comment:
Please don't indent the preprocessor comment, for consistency with later versions:
```java
    exports jdk.internal.logger to
/*[IF OPENJCEPLUS_SUPPORT]*/
        openjceplus,
/*[ENDIF] OPENJCEPLUS_SUPPORT */
```

Reply:
updated